### PR TITLE
staging/dev/options

### PR DESCRIPTION
**Pull request by taylorfturner**
- Updating `dev` with `feature/options` work in preparation for next release
- Main updates through the `feature/options` branch are:
    - New preset option for low memory usage
    -  Row Statistics Options

### CODE REVIEW

**Comment by taylorfturner** [comment]
rebased with `dev`

**Comment by JGSweets** [comment]
Any merge conflicts here @taylorfturner

**Review by JGSweets — APPROVED** [review]

**Comment by taylorfturner** [comment]
> Any merge conflicts here @taylorfturner

Nope -- no conflicts that I resolved to call out on this PR  @JGSweets

File: requirements.txt [hunk 1]
```diff
@@ -10,7 +10,7 @@ fastavro>=1.0.0.post1
 python-snappy>=0.5.4
 charset-normalizer>=1.3.6
 psutil>=4.0.0
-scipy>=1.4.1
+scipy>=1.4.1,<1.11.0
```
Comment (taylorfturner) [inline requirements.txt:13]: fixing scipy issue @JGSweets

**Review by ksneab7 — APPROVED** [review]

**Comment by taylorfturner** [comment]
Documentation update relative to this PR is #920

**Pull request merged**